plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMysqlWithoutDataSource.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.mysql;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.Session;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.DistributedQueryRunner;
+import io.trino.testing.QueryRunner;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static io.airlift.testing.Closeables.closeAllSuppress;
+import static io.trino.testing.TestingSession.testSessionBuilder;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class TestMysqlWithoutDataSource
+        extends AbstractTestQueryFramework
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        Map<String, String> properties = ImmutableMap.<String, String>builder()
+                .put("connection-url", "jdbc:mysql://not_existing_mysql")
+                .put("connection-user", "user")
+                .put("connection-password", "password")
+                .build();
+
+        Session session = testSessionBuilder()
+                .setCatalog("mysql")
+                .build();
+
+        DistributedQueryRunner queryRunner = null;
+        try {
+            queryRunner = DistributedQueryRunner.builder(session).build();
```

Comment:
Do we really need a DistributedQueryRunner here?
This looks like a unit test of MySqlClient after all

Reply:
I wanted to see end-to-end how it works. Testing just MySql is not enough as I don't know how given exception will be handled. 

I don't like LocalQueryRunner.

I can use DistributedQueryRunner with single node.

If DistributedQueryRunner is no go, then I don't see a point to add any tests here.

Reply:
fine by me without a test --  we have
```
catch (SQLException e) {
            throw new TrinoException(JDBC_ERROR, e);
```

in many places in baseJdbcClient s this is just "the pattern"